.github/workflows/build-schedule.yaml

```diff
@@ -0,0 +1,26 @@
+---
+name: Build and Publish Images (scheduled)
+on:
+  schedule:
+    - cron: '0 3 * * *' # run before e2e-nightly to ensure a fresh operator build
+
+jobs:
+  build-operator-index:
+    name: "Trigger build and publish for quay operator"
+    runs-on: ubuntu-latest
+    env:
+      BRANCH: redhat-3.6
+    steps:
+    - uses: convictional/trigger-workflow-and-wait@v1.3.0
+      with:
+        owner: ${{ github.repository_owner }}
+        repo: 'quay-operator'
+        # see https://docs.github.com/en/actions/security-guides/automatic-token-authentication
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        workflow_file_name: build-and-publish.yaml
+        ref: ${{ env.BRANCH }}
+        wait_interval: 30
+        inputs: |
+          {
+            "branch": "${{ env.BRANCH }}"
+          }
```

Comment:
What do you think about creating two other triggers ? I mean, one for `redhat-3.7` and another for `redhat-3.8` (assuming that we are already keeping the `redhat-3.8` in sync with `master`) ?

Reply:
Good idea, will do!

Reply:
Done, PTAL